ui/admin/tests/acceptance/scopes/selectors.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const SCOPE_FORM = 'main .rose-form';
+export const FIELD_STORAGE_POLICY_DROPDOWN = '[name=policy_id]';
```

Comment:
None blocking comment: I think, since this is a form field [using a select](https://helios.hashicorp.design/components/form/select), [rather than a dropdown](https://helios.hashicorp.design/components/dropdown) component, using `_SELECT` rather than `_DROPDOWN` could be more appropriate.

What are your thoughts?